### PR TITLE
BETA: Register shock.is-a.dev

### PR DESCRIPTION
**Pull request by shockbs**
Added `shock.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by satr14washere — APPROVED** [review]

File: domains/pherngpy.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "shockbs",
+        "email": "apipherng@gmail.com"
+    },
+    "record": {
+        "CNAME": "willchangelater"
```
Comment (SX-9) [inline domains/pherngpy.json:7]: placeholders are not allowed

**Review by satr14washere — CHANGES_REQUESTED** [review]

File: domains/pherngpy.json (hunk 1 above)
Comment (shockbs) [reply domains/pherngpy.json:7]: sorry my project aren't ready, i will keep changing servers for this domain so i want to ask my two domains are approved, where should i configure it? it's not showing at manage.is-a.dev

> placeholders are not allowed

Comment (shockbs) [reply domains/pherngpy.json]: > placeholders are not allowed

done

**Review by satr14washere — APPROVED** [review]
looks good

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**